parsec/core/types/pki.py

```diff
@@ -91,9 +91,22 @@ class SCHEMA_CLS(BaseSchema):
         ciphertext = fields.Bytes(required=True)  # An encrypted PendingDeviceKeys
 
         @post_load
-        def make_obj(self, data):
+        def make_obj(
+            self,
+            data: Dict[
+                str,
+                Union[
+                    X509Certificate,
+                    BackendPkiEnrollmentAddr,
+                    DateTime,
+                    UUID,
+                    PkiEnrollmentSubmitPayload,
+                    bytes,
+                ],
+            ],
```

Comment:
Maybe `Dict[str, object]` is enough since you're using a `type: ignore` afterwards?

In general I think it makes sense to use `Dict[str, object]` to type those kind of arbitrary data structure, and then do something like this if necessary:
```python
x509_certificate  = typing.cast(X509Certificate, data["x509_certificate"])
```